portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,12 +34,23 @@
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final String COMMENT_EMAIL = "Email";
   private static final String COMMENT_KIND = "Comment";
   private static final String COMMENT_TIMESTAMP = "timestamp";
   private static final String CONTENT_TYPE = "text/html;";
   private static final String COMMENT_VALUE = "text";
   private static final String DEFAULT_MAX_COMMENTS = "20";
 
+  private class Comment {
```

Comment:
I made a dummy example using protobufs. Take a look? It basically does the same thing you're doing now, just that it also conveniently generates getter/setter methods for you.
https://github.com/chen-dawn/step/pull/6